app/lib/three_scale/json_validator.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  class JSONValidator < ::JSON::Validator
+    def self.setup!
+      self.schema_reader = JSON::Schema::Reader.new(accept_uri: false, accept_file: false)
```

Comment:
The JSON::Validator is defining it as a class variable which is not right IMHO

Is it a way to have it as an instance, we might use it in Swagger and in policies?

Reply:
It turns out we were using the same options here and in `Swagger`, that's why I kept it.